src/osdep/win32/overrides.c

```diff
@@ -664,7 +670,7 @@ int win32_select (int num_fds, struct fd_set *rd, struct fd_set *wr,
 		}
 	}
 
-	errno = 0;
+	int errnol = 0;
```

Comment:
Where this variable is being changed?

Reply:
It's tru that this variable has to be reviewed. It just didn't work with errno. I'll explore some more.